docs/resources/aws_iam_group.md

```diff
@@ -31,12 +31,13 @@ See also the [AWS documentation on IAM Groups](https://docs.aws.amazon.com/IAM/l
 
 ## Properties
 
-|Property    | Description|
-| ---        | --- |
-|group\_name | The group name. |
-|group\_id   | The group ID. |
-|arn         | The Amazon Resource Name of the group. |
-|users       | Array of users associated with the group.  |
+|Property            | Description|
+| ---                | --- |
+|group\_name         | The group name. |
+|group\_id           | The group ID. |
+|arn                 | The Amazon Resource Name of the group. |
+|users               | Array of users associated with the group.  |
+|inline_policy_names | The arns of the standalone IAM policies which are attached to the user. |
```

Comment:
Is this the policy name or the ARN?

Reply:
The arn returns the Amazon Resource Name (ARN) for the specified AWS::IAM::Group resource. For example: arn:aws:iam::123456789012:group/mystack-mygroup-1DZETITOWEKVO.

Source: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-iam-group.html

Reply:
@IanMadd, to summarise, yes it is just the policy name.

Reply:
Good catch, will change